### PR TITLE
Updated readme

### PR DESCRIPTION
**Pull request by brittanywang**
Links and screenshots should be correct now. LMK if there's something that I missed!

### CODE REVIEW

**Comment by nkadonaga** [comment]
Awesome job. Can we update the security section since we now have a serverless backend?

You could probably copy some text from Jon's phase 3 project plan:

> We decided to use AWS Lambda, a serverless computing platform that allows you to run Python code. The benefits are twofold. First, AWS Lambda takes care of everything required to run and scale our code. Second, its built-in sandbox let's you safely run code written in a handful of languages. This allows us to have a serverless backend, where there is no need for our team to manage or maintain a server. In addition, AWS Lambda is priced by code execution time so nothing is charged when your code isn't running.

Alternatively, we might delete the section altogether, since it might be safer to not list the security flaws (haha)?

File: README.md [hunk 1]
```diff
@@ -43,7 +43,4 @@ See the [Issues](https://github.com/cs121-team-panda/coconut-interpreter-flask/i
 
 ## Security
 
-Major security concerns for online compilers/interpreters are malicious codes submitted from users. In our current system, the code is evaluated on the server-side. From the server perspective, the server may happen to run the potentially malicious code. Some potential problems may arise, at least but not limited to:
-
-* DDoS attacks to exhaust the server's resources.
-* Manipulation of file structures using `os` module
+The Coconut Interpreter uses AWS Lambda, a containerized, serverless backend which allows users to run code safely.
```
Comment (nkadonaga) [inline README.md]: Maybe instead of "which allows users..." do "which allows for safe execution of untrusted code"?

**Comment by jcruz** [comment]
One thing to note which shouldn't affect this PR is that our screenshots represent the react app not the flask app.

File: README.md [hunk 2]
```diff
@@ -6,35 +6,51 @@ Try the current version at: [https://cs121-team-panda.github.io/coconut-interpre
 
 Developed by Jonathan Cruz, Teerapat Jenrungrot, Natalie Kadonaga, and Brittany Wang.
 
+## Installation
+```pip install -r requirements.txt```
```
Comment (jcruz) [inline README.md]: nit: change to
````md
```
pip install -r requirements.txt
```
````

File: README.md [hunk 3]
```diff
@@ -6,35 +6,51 @@ Try the current version at: [https://cs121-team-panda.github.io/coconut-interpre
...
+
+## Running the app
+Requires Python 3 [(recent Python version here)](https://www.python.org/downloads/). 
+```
+git clone https://github.com/cs121-team-panda/coconut-interpreter-flask.git
```
Comment (jcruz) [inline README.md]: nit: let's remove git clone

File: README.md [hunk 4]
```diff
@@ -6,35 +6,51 @@ Try the current version at: [https://cs121-team-panda.github.io/coconut-interpre
 
 Developed by Jonathan Cruz, Teerapat Jenrungrot, Natalie Kadonaga, and Brittany Wang.
 
+## Installation
+```pip install -r requirements.txt```
+
+## Running the app
+Requires Python 3 [(recent Python version here)](https://www.python.org/downloads/).
```
Comment (jcruz) [inline README.md]: nit: Up to you but we don't really need to link to Python since people get it from other sources such as brew and Anaconda

**Pull request merged**